src/components/SVGIcon.vue

```diff
@@ -13,13 +13,32 @@
     aria-hidden="true"
     class="svg-icon"
     xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
   >
-    <slot />
+    <use v-if="themeOverrideURL" :href="`${themeOverrideURL}#${themeId}`" />
```

Comment:
👏 very good

Reply:
This still seems pretty magical to me. The documentation will need to clearly point out the importance of using the right `id` attribute in the overriding SVG file, because this magic doesn't work without that 😢

It is pretty cool though, because it allows us to both reference and embed the SVGs, allowing for the app to still control fill colors and things.

Reply:
I think if there are no drawbacks of this, it might make sense in the future to actually extract the icons back into SVGs, as its much easier to work with and preview them. Just a thought.

Reply:
Hm, I like that thought since it would be nice to have the raw SVG assets isolated from the actual Vue components.

Reply:
Added docs for this to the DocC PR.